File: blaster-ui/src/app/model/donor.ts
import { BloodGroup } from "./enums";
import { Donation } from "./donation";

export class Donor {
  public _id: string;
  public _rev: string;
  public donorAadhar: string;
  public donorName: string;
  public bloodGroup: BloodGroup;
  public donations: Donation[];
}

File: server-connector/src/main/java/com/redavatar/serverconnector/model/blockchain/Donor.java
package com.redavatar.serverconnector.model.blockchain;

import java.util.List;

import com.redavatar.serverconnector.model.BloodGroup;
import com.redavatar.serverconnector.model.DonorStatus;

import lombok.Data;

@Data
public class Donor {
  private String donorId;
  private String donorName;
  private BloodGroup bloodGroup;
  private DonorStatus donorStatus;
  private String donorMobileNumber;
  private String donorEmail;
  private List<Donation> donationDetails;
}